Dockerfiles/gadget-default.Dockerfile

```diff
@@ -43,13 +48,24 @@ RUN cd /gadget/gadget-container && \
 
 FROM bcc
 
+# This will be used by dlv server.
+ARG DEBUG_PORT
+ENV DEBUG_PORT=${DEBUG_PORT}
+
 RUN set -ex; \
 	export DEBIAN_FRONTEND=noninteractive; \
 	apt-get update && \
 	apt-get install -y --no-install-recommends \
 		ca-certificates jq wget xz-utils binutils && \
 		rmdir /usr/src && ln -sf /host/usr/src /usr/src && \
-		rm -f /etc/localtime && ln -sf /host/etc/localtime /etc/localtime
+		rm -f /etc/localtime && ln -sf /host/etc/localtime /etc/localtime; \
+	if [ ${DEBUG_PORT} ]; then \
```

Comment:
We can also have a look at the idea of  [`target`](https://docs.docker.com/build/building/multi-stage/#stop-at-a-specific-build-stage). Moving these in the end as `dev` stage and if target isn't configured we build a debug image and when we specify a target like `prod` we stop without doing any debug stuff e.g I got this idea from [here](https://github.com/cilium/cilium/blob/master/images/cilium/Dockerfile#L75). Just an idea, what do you think about it?